### ISSUE TITLE
BOTTOM_SHEET doesn't work

### ISSUE THREAD

**Issue by campio97** [salim-lachdhaf/searchable_dropdown#366]
Hello, same issue here.
```
return DropdownSearch<Stati>(
      mode: Mode.BOTTOM_SHEET, //Workd with dialog and menu, but no with BOTTOM_SHETT
      selectedItem: new Stati(id: 106, nome: "Italia"),
      isFilteredOnline: false,
      items: this.stati,
      showSearchBox: true,
      itemAsString: (Stati s) => s.nome,
      popupTitle: Text(
        ApplicationLocalizations.of(context).translate('cerca'),
        textAlign: TextAlign.center,
        style: TextStyle(fontWeight: FontWeight.bold),
      ),
      dropdownSearchDecoration: InputDecoration(
        labelText: ApplicationLocalizations.of(context).translate('stato'),
        labelStyle: TextStyle(fontWeight: FontWeight.bold),
        icon: Icon(
          Icons.search,
          color: Color.fromRGBO(91, 151, 181, 1),
        ),
      ),
      onChanged: (Stati data) {
        print(data);
      },
    );
```

if I do the search the list is not updated, moreover I have noticed that if I do a search (the list is not updated) and if I click on an element in the list it is not selected

_Originally posted by @campio97 in https://github.com/salim-lachdhaf/searchable_dropdown/issues/358#issuecomment-1031384882_

**Comment by shamsmm** [comment]
I have the same issue, it is very buggy and rarely works

**Comment by vasilich6107** [comment]
@campio97 your code sample is not runnable
Where should I get Stati object?

**Comment by vasilich6107** [comment]
@supratikkoley @krasznaidavid @shamsmm 

could you provide copy pastable reproduction code which will show your issue?
Something like main.dart

**Comment by salim-lachdhaf** [comment]
@campio97 @supratikkoley @shamsmm @krasznaidavid Tests done on the last version and there is no issue,
Could you provide a pastable reproduction code ? or the issue will be closed

**Comment by Raviteja11122** [comment]
same here 
`DropdownSearch<String>.multiSelection(
      mode: Mode.BOTTOM_SHEET,
      showSelectedItems: true,
      showSearchBox: true,
      items: ["Brazil", "Italia", "Tunisia", 'Canada', "India", "China"],
      onChanged: print,
      selectedItems: ["Brazil"])`
      
      


https://user-images.githubusercontent.com/49988053/163832410-c063bc56-e795-4547-9d85-abd7777a62e3.mp4

**Comment by Raviteja11122** [comment]
same here 
```
DropdownSearch<String>.multiSelection(
      mode: Mode.BOTTOM_SHEET,
      showSelectedItems: true,
      showSearchBox: true,
      items: ["Brazil", "Italia", "Tunisia", 'Canada', "India", "China"],
      onChanged: print,
      selectedItems: ["Brazil"])
      ```

https://user-images.githubusercontent.com/49988053/163832876-f7c732c3-d963-42a7-a157-37e822dd5e1c.mp4

**Comment by Raviteja11122** [comment]
This is fixed in v3.0.1

**Comment by krasznaidavid** [comment]
I confirm, this is fixed in v3.0.1
